TaskfMRIAnalysis/scripts/makeSubjectTaskSummary.sh

```diff
@@ -0,0 +1,363 @@
+opts_AddOptional '--parcellationfile' 'ParcellationFile' '/path/to/dlabel' "Absolute path to the parcellation dlabel file. Default=NONE" 'NONE'
+
+opts_ParseArguments "$@"
+
+# if LevelOnefsfNames is blank, set equal to LevelOnefMRINames
+[ -z "$LevelOnefsfNames" ] && LevelOnefsfNames=${LevelOnefMRINames}
+# if LevelTwofsfName is blank, set equal to LevelTwofMRIName
+[ -z "$LevelTwofsfName" ] && LevelTwofsfName=${LevelTwofMRIName}
+
+#display the parsed/default values
+opts_ShowValues
+
+
+# ------------------------------------------------------------------------------
+#  Verify required environment variables are set and log value
+# ------------------------------------------------------------------------------
+
+if ((pipedirguessed))
+then
+    log_Err_Abort "HCPPIPEDIR is not set, you must first source your edited copy of Examples/Scripts/SetUpHCPPipeline.sh"
+fi
+
+log_Msg "Platform Information Follows: "
+uname -a
+
+opts_ShowVersionIfRequested $@
+${HCPPIPEDIR}/show_version
+
+log_Check_Env_Var HCPPIPEDIR
+log_Check_Env_Var FSLDIR
+log_Check_Env_Var CARET7DIR
+
+HCPPIPEDIR_tfMRIAnalysis=${HCPPIPEDIR}/TaskfMRIAnalysis/scripts
+
+
+##### SET_NAME_STRINGS: smoothing and filtering string variables used for file naming #####
+SmoothingString="_s${FinalSmoothingFWHM}"
+
+# Record prior preprocessing in filenames
+if [ "${ProcSTRING}" != "NONE" ] ; then
+	ProcSTRING="_${ProcSTRING}"
+else
+	ProcSTRING=""
+fi
+
+# Record additional highpass filter in filenames
+if [ "${TemporalFilter}" != "NONE" ]; then
+	TemporalFilterString="_hp""$TemporalFilter"
+else
+	TemporalFilterString="_hp0"
+fi
+
+# Record additional lowpass filter in filenames
+if [ "${TemporalSmoothing}" != "NONE" ]; then
+	LowPassSTRING="_lp""$TemporalSmoothing"
+else
+	LowPassSTRING=""
+fi
+
+# Set variables used for different registration procedures
+if [ "${RegName}" != "NONE" ] ; then
+	RegString="_${RegName}"
+else
+	RegString=""
+fi
+
+if [[ "${SummaryName}" = "NONE" || "${SummaryName}" = "" ]]; then
+	if [ "${LevelTwofMRIName}" = "NONE" ]; then
+	    log_Err_Abort "Cannot determine summaryname. You must provide name for single-subject summary directory if Level2 analysis is not being run."
+	else
+		SummaryName="${LevelTwofMRIName}/${LevelTwofsfName}";
+	fi
+fi
+
+log_Msg "MAIN: SET_NAME_STRINGS: SmoothingString: ${SmoothingString}"
+log_Msg "MAIN: SET_NAME_STRINGS: TemporalFilterString: ${TemporalFilterString}"
+log_Msg "MAIN: SET_NAME_STRINGS: RegString: ${RegString}"
+
+
+# Determine locations of necessary directories (using expected naming convention)
+ResultsFolder="${Path}/${Subject}/MNINonLinear/Results"
+
+# Determine which analyses need to be packaged into summary directory
+# initialize run variables
+runParcellated=false; runVolume=false; runDense=false;
```

Comment:
These variables don't appear to get tested, merely set and then ignored.

Reply:
These are removed in next commit.